requirements-dev.txt

```diff
@@ -2,11 +2,12 @@ shortuuid
 werkzeug
 django_pdb
 python-dateutil
+pytest<8
 pytest-django
 pytest-cov
 factory-boy
 requests
-pygments
+pygments<2.18
```

Comment:
Something changed between Pygment 2.17 and 2.19 (installed on Python 3.8) which changes `HighlightTagTests.test_should_highlight_python_syntax_with_name`. 

It seems that Pygment 2.18+ doesn't support Python 3.7, so we get a different output in the test depending on the Python version...

As Python 3.7 is EOL, next step would be to drop support for it, unpin this version and update the test to match the newer output.

Reply:
> As Python 3.7 is EOL, next step would be to drop support for it, unpin this version and update the test to match the newer output.

Next step here: https://github.com/django-extensions/django-extensions/pull/1910 (includes some changes from this PR)